src/pages/files/index.tsx

```diff
@@ -38,131 +38,112 @@ export default function Files(): ReactElement {
 
   const [inputMode, setInputMode] = useState<'browse' | 'upload'>('browse')
   const [searchInput, setSearchInput] = useState('')
```

Comment:
I am not sure that `searchInput` is the best name for this, maybe `hashInput` or `referenceInput` is more appropriate.